### DISCUSSION TITLE
Masque not Supporting MoveAnything

### DISCUSSION THREAD

**Discussion by scidrov** [SFX-WoW/Masque#170] [Issues] [answered]
well i have a custom UI where i resized everything and moved somewhere else via moveanything and installed Masque the the buttons dont have the border

**Comment by StormFX** [comment]
First, Masque does not support any add-ons. The add-ons have to support Masque.  Second, what add-on are you using for the action buttons? Masque does not support the default buttons, as is stated on in the description.

**Comment by scidrov** [comment]
no addons i just move the original action bars around a little and resized them and disabled the eagles ( ._.)

**Comment by StormFX** [comment, marked as answer]
Ah, well, you have to have a button add-on installed to use Masque.
